.github/actions/check/action.yaml

```diff
@@ -0,0 +1,21 @@
+name: Check
+description: 'This action runs the Terraform fmt and validate commands'
```

Comment:
See if you can set the `TF_IN_AUTOMATION` env variable.

Reply:
I add